ansible-scylla-node/tasks/RedHat.yml

```diff
@@ -30,7 +30,37 @@
       yum:
         name: kernel-ml
         state: present
-  when: elrepo_kernel == True and install_type == 'online' and (ansible_distribution == "CentOS" or ansible_distribution == "RedHat")
+
+    - name: Configure grub2 to boot first entry
+      lineinfile:
+        dest: /etc/default/grub
+        owner: root
+        group: root
+        mode: '0644'
+        backup: yes
+        create: yes
+        regexp: '{{ item.regexp }}'
+        line: '{{ item.line }}'
+        state: present
+      with_items:
+        - regexp: 'GRUB_DEFAULT='
+          line: 'GRUB_DEFAULT=0'
+        - regexp: 'GRUB_TIMEOUT='
+          line: 'GRUB_TIMEOUT=30'
```

Comment:
Not sure such a long timeout is required. @vladzcloudius @tarzanek wdyt?

Reply:
I agree. But why touching the timeout config in the first place?